spec/metrician_spec.rb

```diff
@@ -271,6 +271,28 @@ def app
       end
     end
 
+    describe "middleware exceptions" do
+      def app
+        require "metrician/middleware/request_timing"
+        require "metrician/middleware/application_timing"
+        Rack::Builder.app do
+          use Metrician::Middleware::RequestTiming
+          use Metrician::Middleware::ApplicationTiming
+          run lambda { |env| raise "boom" }
+        end
+      end
+
+      specify "middleware exceptions don't cause errors in response size tracking" do
+        Metrician.configuration[:request_timing][:response_size][:enabled] = true
```

Comment:
I been thinking about how this effects downstream tests, and I think we should have a thing in our test setup that resets the configuration to default after every test.